test/actions/FilterActionsTest.js

```diff
@@ -0,0 +1,180 @@
+'use strict'
+import { expect } from 'chai'
+import queryString from 'query-string'
+import * as filterActions from '../../src/actions/filterActions'
+
+describe('FilterActions', function () {
+    describe('getFilterFromUrl', function() {
```

Comment:
I want to pair with you on testing, there are some good opportunities to improve clarity  in your testing strategy.